docs/applications/advanced/custom-dependencies.md

```diff
@@ -15,12 +15,13 @@ Cortex looks for a requirements.txt file in the top level directory of the app (
 └── resources/
 ```
 
-## Your code
+## Custom Python Packages
 
 Cortex can install your Python package(s) to be made available in all environments.
 
 Cortex looks for your Python packages in the directory `./packages/<package name>`. The package must have a `setup.py` in the root of the package directory with the name set to your package name. Cortex will run `pip3 wheel -w wheelhouse ./packages/<package name>` to construct wheels for Python Project. These wheels will be installed for your use.
 
+Cortex App Directory
 ```text
```

Comment:
Some markdown viewers do weird things when there isn't a new line directly above a code block. Can you add one here and below?